virttest/utils_hotplug.py

```diff
@@ -364,6 +364,55 @@ def check_xmlcount(vm, exp_vcpu, option):
     return result
 
 
+def get_cpustats(vm, cpu=None):
+    """
+    Get the cpustats output of a given domain
+    :param vm: VM domain
+    :param cpu: Host cpu index, default all cpus
+    :return: dict of cpu stats values
+    result format:
+    {0:[vcputime,emulatortime,cputime]
+    ..
+    'total':[cputime]}
+     """
+    host_cpu_online = utils.cpu_online_list()
+    cpustats = {}
+    if cpu:
+        cpustats[cpu] = []
+        option = "--start %s --count 1" % cpu
+        result = virsh.cpu_stats(vm.name, option)
+        output = result.stdout.strip().split()
```

Comment:
It will be better if we can retrieve output after we check for status in next line.

Reply:
sure